### DISCUSSION TITLE
Use Phi-3 to create a VSCode chat agent extension: Session Resources

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#69] [Announcements]
This is the resources and discussion thread for this live stream: https://aka.ms/raghack/phi3
It will be updated with links after the session.

Recording: https://www.youtube.com/watch?v=dSvhHOMSq5Y
Slides: [PPT](https://github.com/microsoft/RAG_Hack/blob/main/slides/RAGHackSession-VSCodeAgentPhi3.pptx)

Please ask any follow-up questions here, and the speakers will see them.

**Comment by reza8iucs** [comment]
Hi where can i find the resources for this demo?

**Reply by multispark** [reply]
Hello, please take a look at the slides!
